pkg/apis/contrail/v1alpha1/status.go

```diff
@@ -0,0 +1,25 @@
+package v1alpha1
+
+import appsv1 "k8s.io/api/apps/v1"
+
+// Status is the status of the service.
+// +k8s:openapi-gen=true
+type Status struct {
+	Active        bool  `json:"active,omitempty"`
+	Replicas      int32 `json:"replicas,omitempty"`
+	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
+}
```

Comment:
Is it going to be used in other resources apart from memcached?

Reply:
The plan is to use for all resources. We plan to adjust other resources one by one

Reply:
Great :+1: